### PR TITLE
Add highlighter toggle effect

### PR DESCRIPTION
**Pull request by rish987**
I often switch between the highlighter and fineliner, so I thought it would be useful to have an effect that toggles between them (like we can toggle between eraser and pen).

### CODE REVIEW

File: sources/effects.c [hunk 1]
```diff
@@ -85,7 +85,7 @@ int write_oriented_tap_sequence(int fd, int numLocations, ...) {
     if (action == SLEEP) {
       usleep(50000);
     } else if (action == LONG_SLEEP) {
-      usleep(200000);
+      usleep(300000);
```
Comment (slotThe) [inline sources/effects.c:88]: I guess you tested it with `200000` and it was a bit flaky?

**Review by slotThe — COMMENTED** [review]
Thank you for the contribution :tada: Sorry that I've been a bit slow with reviewing this (Christmas, life, and so on :) but I'm slowly getting to them!

File: sources/effects.c (hunk 1 above)
Comment (rish987) [reply sources/effects.c:88]: Yeah. What do you think about making this configurable?

Comment (slotThe) [reply sources/effects.c:88]: Perhaps that's the best solution, yeah

Comment (slotThe) [reply sources/effects.c:88]: Perhaps that's the best solution, yeah

Comment (rish987) [reply sources/effects.c]: I've reverted this change for now.

**Comment by slotThe** [comment]
Thanks!

**Pull request merged**